pkg/csi_driver/node.go

```diff
@@ -157,13 +183,24 @@ func validateVolumeAttributes(attr map[string]string) error {
 	return nil
 }
 
+func validateNodePublishSecrets(secrets map[string]string) error {
```

Comment:
This is specific to SMB. Make that explicit in the function name.

Reply:
Is `validateSmbNodePublishSecrets` okay for the name?